src/main/resources/com/google/api/codegen/nodejs/sample.snip

```diff
@@ -62,10 +62,14 @@
     @end
     @if class.hasRequestBody
       resource: {
-        @if class.methodVerb == "PATCH"
-          // TODO: Add desired properties of the request body. Only added properties
+        @switch class.methodVerb
+        @case "PATCH"
+          // TODO: Add desired properties of the request body. Only these properties
```

Comment:
Hmmm, I think "Assign values..." would work just as well, but OK.

If keeping as you have it, how about s/of the request body/to the request body/g?